backend/agent/ai_model.py

```
@@ -0,0 +1,18 @@
+from pydantic_ai import Agent, RunContext
```

Ruff (0.11.9)

`pydantic_ai.Agent` imported but unused

Remove unused import

(F401)